### PR TITLE
Add note about installer to readme

### PR DESCRIPTION
**Pull request by johnmcollier**
Adds a small notice to the repo's readme to point to the installer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #  AI-lab Software Templates
 
+> [!IMPORTANT]  
+> The software templates in this repository require OpenShift Pipelines and OpenShift GitOps installed, alongside their respective plugins in Red Hat Developer Hub. For more information on configuring them, please see [the installer we have prepared](https://github.com/redhat-ai-dev/ai-rhdh-installer/).
```
Comment (thepetk) [inline README.md:4]: Tiny nit, it's totally ok for me with the extisting content. I only think better to use the `ai-rhdh-installer` "branding".

```suggestion
> The software templates in this repository require OpenShift Pipelines and OpenShift GitOps installed, alongside their respective plugins in Red Hat Developer Hub. For more information on configuring them, please see the [ai-rhdh-installer](https://github.com/redhat-ai-dev/ai-rhdh-installer/).
```

Comment (Jdubrick) [inline README.md:4]:
```suggestion
> The software templates in this repository require OpenShift Pipelines and OpenShift GitOps installed, along with their respective plugins in Red Hat Developer Hub. For more information on configuring them, please see [the installer we have prepared](https://github.com/redhat-ai-dev/ai-rhdh-installer/).
```

I'm good with Fanis' nit, personally I think if the comma is there we break apart `alongside`, wdyt?

**Review by maysunfaisal — DISMISSED** [review]
lgtm after addressing other comments

**Comment by maysunfaisal** [comment]
@johnmcollier also need to update the release branch, so open another PR  from the same branch

File: README.md (hunk 1 above)
Comment (johnmcollier) [reply README.md]: Updated.

Comment (johnmcollier) [reply README.md]: Updated.

**Review by Jdubrick — APPROVED** [review]

**Pull request merged**